google/cloud/storage/benchmarks/aggregate_throughput_benchmark.cc

```diff
@@ -215,10 +218,12 @@ int main(int argc, char* argv[]) {
   using clock = std::chrono::steady_clock;
   auto const start = clock::now();
   auto const deadline = clock::now() + options->running_time;
```

Comment:
Off topic, but `s/clock::now()/start/` perhaps?

But then again, perhaps all the `clock::now()`/`start`/`deadline` stuff should disappear in favor of `timer.Sample().elapsed_time`.